Amoraitis.TodoList/Controllers/ManageController.cs

```diff
@@ -516,6 +444,18 @@ private string FormatKey(string unformattedKey)
             return result.ToString().ToLowerInvariant();
         }
 
+        private async Task<ApplicationUser> GetUserOrFail()
```

Comment:
I love this new method to handle failures, when we are trying to get the current user! A better way to implement that is to `return Challenge;` instead of throwing an exception. Currently, we are not able to implement that, because we need a `Logger`. Let's discuss it in a new issue...

Reply:
Excellent, I tried to not change the implementation but this was something useful. 😅

If we can do better then we should.